src/rt-app.c

```diff
@@ -861,8 +856,12 @@ static void set_thread_affinity(thread_data_t *data, cpuset_data_t *cpu_data)
 	if (actual_cpu_data->cpuset == NULL)
 		actual_cpu_data = &data->def_cpu_data;
 
-	if (!CPU_EQUAL(actual_cpu_data->cpuset, data->curr_cpu_data->cpuset))
-	{
+	if (
+	    actual_cpu_data->cpusetsize != data->curr_cpu_data->cpusetsize ||
```

Comment:
Witht this PR, cpusetsize is always set to sizeof(cpu_set_t) so this test is useless